fossunited/ticketing/doctype/foss_event_ticket/foss_event_ticket.py

```diff
@@ -35,6 +35,7 @@ class FOSSEventTicket(Document):
         organization: DF.Data | None
         razorpay_payment: DF.Link | None
         tier: DF.Data | None
+        tshirt_assigned: DF.Check
```

Comment:
By assigned, do you mean distributed or disbursed or delivered? If a person purchases a t-shirt, I would say that a t-shirt is assigned to the ticket. And separately, it is "delivered" to the participant, whether that delivery happens in-person or otherwise.

Reply:
Ah yes. I can rename it to "tshirt_delivered" if that would make it more clear?